src/index.jsx

```diff
@@ -71,12 +71,22 @@ const App = () => {
             }}
           />
           {process.env.ENABLE_TAGGING_TAXONOMY_PAGES === 'true' && (
-            <Route
-              path="/taxonomy-list"
-            >
-              <TaxonomyListPage />
-            </Route>
+            <>
+              <Route exact path="/taxonomy-list/" component={TaxonomyListPage} />
+              <Route
+                path="/taxonomy-list/:taxonomyId"
+                render={({ match }) => {
+                  const { params: { taxonomyId } } = match;
+                  return (
+                    <TaxonomyDetailPage taxonomyId={Number(taxonomyId)} />
+                  );
+                }}
+              />
+            </>
```

Comment:
There's a routing bug that occurs if you follow these steps:

1. Visit http://localhost:2001/taxonomy-list/   <-- note trailing slash
2. Click on one of the taxonomies there, and note the URL looks like this: `http://localhost:2001/taxonomy-list/taxonomy-list/<pk>`

I'm not sure what the fix is, but I recommend looking at how the other nested routes are written and imitating that.

Reply:
Thank you for getting that!

The react-router was updated (https://github.com/openedx/frontend-app-course-authoring/pull/519) to V6. I will merge this update in the [FAL-3528](https://github.com/open-craft/frontend-app-course-authoring/tree/chris/FAL-3528-taxonomy-export-menu) branch and rebase here before fixing it!

Reply:
Fixed!

Reply:
Excellent, great fix @rpenido !

And I like how you pulled the common header/footer into a single `TaxonomyLayout` wrapper, that's perfect.